### DISCUSSION TITLE
Error creating/adding extensions to database

### DISCUSSION THREAD

**Discussion by shoxadomokai** [supabase/supabase#13077] [Questions]
I am unable to add a new extension to my database.

When using the dashboard; I get the following error: "Failed to toggle [EXTENSION_NAME]: failed to create pg.extensions: Connection terminated unexpectedly" and when using pgAdmin, I get the following error: "ERROR: SSL SYSCALL error: EOF detected"

**Comment by gaslitbytech** [comment]
Is the extension in the list of Supabase pre-configured extensions [here](https://supabase.com/docs/guides/database/extensions)?

**Reply by shoxadomokai** [reply]
Yes it is. I'm trying to install both pg_trgm and fuzzystrmatch.
